src/components/InventoryGroupDetail/GroupTabDetails.js

```diff
@@ -0,0 +1,115 @@
+import { useLocation, useNavigate } from 'react-router-dom';
+import { resolveRelPath } from '../../Utilities/path';
+import {
+  getNotificationProp,
+  manageEdgeInventoryUrlName,
+} from '../../Utilities/edge';
+import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import ErrorState from '@redhat-cloud-services/frontend-components/ErrorState';
+import {
+  Bullseye,
+  PageSection,
+  Spinner,
+  Tab,
+  TabTitleText,
+  Tabs,
+} from '@patternfly/react-core';
+import React, { Suspense, lazy, useState } from 'react';
+import { hybridInventoryTabKeys } from '../../Utilities/constants';
+import GroupSystems from '../GroupSystems';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { usePermissionsWithContext } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
+import { REQUIRED_PERMISSIONS_TO_READ_GROUP_HOSTS } from '../../constants';
+
+import { EmptyStateNoAccessToSystems } from './EmptyStateNoAccess';
+const GroupTabDetailsWrapper = ({ groupId, groupName, activeTab }) => {
+  const GroupDetailInfo = lazy(() => import('./GroupDetailInfo'));
```

Comment:
I think it is better to load this out of the Component scope to avoid unpredictable React behaviours